src/navigation/Routes.android.js

```diff
@@ -3,6 +3,7 @@ import { createStackNavigator } from '@react-navigation/stack';
 import React, { useContext, useMemo } from 'react';
 import { InitialRouteContext } from '../context/initialRoute';
 import AddCashSheet from '../screens/AddCashSheet';
```

Comment:
@osdnk do we want to include ConnectedDappSheet in Android navigation routes as well

Reply:
@osdnk  @jinchung  Just fixed it.  PoW: 
<img width="483" alt="Screen Shot 2021-03-15 at 7 24 59 PM" src="https://user-images.githubusercontent.com/1247834/111234071-2bcef600-85c4-11eb-962c-bb85d042f7f5.png">